Documentation/getting-started.md

```diff
@@ -1,24 +1,32 @@
 ## Getting started
 
-The Adobe Experience Platform Media for Edge Network mobile extension has the following dependencies, which must be installed prior to installing the extension:
-- [AEPCore](https://github.com/adobe/aepsdk-core-ios)
-- [AEPEdge](https://github.com/adobe/aepsdk-edge-ios)
-- [AEPEdgeIdentity](https://github.com/adobe/aepsdk-edgeidentity-ios)
+## Configure the Media for Edge Network extension in the Data Collection UI
```

Comment:
`Configure the Adobe Streaming Media for Edge Network extension`

Since this is the start of the doc and also involves data collection UI